src/Navigation.js

```diff
@@ -1,131 +1,61 @@
-import React, { useContext } from "react";
-import { Navbar, Nav, Card, Accordion, Dropdown } from "react-bootstrap";
+import React, {useContext}  from "react";
+import { Navbar, Nav, Accordion } from "react-bootstrap";
 import { Link } from "react-router-dom"
-import { AuthContext } from "./AuthContext";
-import DropdownMenu from "react-bootstrap/DropdownMenu";
-import Logo from "./assets/images/AnitaBLogo.png"
 import './Nav.css'
+import { AuthContext } from "./AuthContext";
 
 
 export default function Navigation() {
-    const { user, isAuth, login, logout } = useContext(AuthContext);
-
+    const { user, isAuth, login, logout } = useContext(AuthContext)
```

Comment:
Sorry, @stableapple , I missed this point before. Here we only use isAuth, login and logout. user is not needed. Thanks.